lib/collection/src/shard/shard_holder.rs

```diff
@@ -21,7 +22,8 @@ impl ShardHolder {
     pub fn new(hashring: HashRing<ShardId>) -> Self {
         Self {
             shards: HashMap::new(),
-            shard_transfers: HashMap::new(),
+            shard_transfers: SaveOnDisk::load_or_init("shard_transfers")
```

Comment:
Are all `shard_transfers` sharing the same path?
I would expect it to be persisted inside `storage/collections/$collection-name/`

Reply:
Good catch, I forgot to add the collection path

Reply:
Fixed